pkg/security/ebpf/c/flow.h

```diff
@@ -34,28 +34,56 @@ __attribute__((always_inline)) u32 get_flow_pid(struct pid_route_t *key) {
     return *value;
 }
 
+__attribute__((always_inline)) u16 get_family_from_sock_common(struct sock_common *sk) {
+    u64 sock_common_skc_family_offset;
+    LOAD_CONSTANT("sock_common_skc_family_offset", sock_common_skc_family_offset);
+
+    u16 family;
+    bpf_probe_read(&family, sizeof(family), (void*)sk + sock_common_skc_family_offset);
+    return family;
+}
+
+__attribute__((always_inline)) u64 get_flowi4_saddr_offset() {
```

Comment:
doesn't seem to be used

Reply:
That's because I made a mistake ! Thank you for noticing 👍🏻